### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/RingTheory/Binomial.lean
Comment (grunweg) [inline Mathlib/RingTheory/Binomial.lean:409]: Nice golf!

File: Mathlib/RingTheory/MvPowerSeries/NoZeroDivisors.lean
Comment (grunweg) [inline Mathlib/RingTheory/MvPowerSeries/NoZeroDivisors.lean:59]: I'm not sure if I understand this change: let's have another pair of eyes on it.

**Review by grunweg — COMMENTED** [review]
Looks very nice, as usual - thanks!.
One change I couldn't quite understand, perhaps another reviewer or maintainer does:
maintainer merge

File: Mathlib/RingTheory/MvPowerSeries/NoZeroDivisors.lean
Comment (riccardobrasca) [reply Mathlib/RingTheory/MvPowerSeries/NoZeroDivisors.lean:59]: It doesn't introduce a new hypothesis, why not.

**Review by riccardobrasca — APPROVED** [review]
Thanks!

bors merge

**Pull request closed without merging**
